src/components/AvatarWithDisplayName.js

```diff
@@ -41,16 +44,32 @@ const defaultProps = {
     personalDetails: {},
     policies: {},
     report: null,
+    isAnonymous: false,
     size: CONST.AVATAR_SIZE.DEFAULT,
 };
 
 const AvatarWithDisplayName = (props) => {
-    const title = ReportUtils.getDisplayNameForParticipant(props.report.ownerEmail, true);
+    const title = props.isAnonymous ? props.report.displayName : ReportUtils.getDisplayNameForParticipant(props.report.ownerEmail, true);
     const subtitle = ReportUtils.getChatRoomSubtitle(props.report);
     const isExpenseReport = ReportUtils.isExpenseReport(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
     const ownerPersonalDetails = OptionsListUtils.getPersonalDetailsForLogins([props.report.ownerEmail], props.personalDetails);
     const displayNamesWithTooltips = ReportUtils.getDisplayNamesWithTooltips(ownerPersonalDetails, false);
+
+    let avatarContainerStyle = styles.emptyAvatar;
+
+    if (props.size === CONST.AVATAR_SIZE.SMALL) {
+        avatarContainerStyle = styles.emptyAvatarSmall;
+    }
+
+    if (props.size === CONST.AVATAR_SIZE.MEDIUM) {
+        avatarContainerStyle = styles.emptyAvatarMedium;
+    }
+
+    if (props.size === CONST.AVATAR_SIZE.LARGE) {
+        avatarContainerStyle = styles.emptyAvatarLarge;
+    }
```

Comment:
Can we do something similar to [this](https://github.com/Expensify/App/blob/109421eb954a76c78db04abe1e4ac216a433d94c/src/styles/StyleUtils.js#L32-L43) and declare a single style that takes in the size variable?

Reply:
Sure, updated